runtime/cudaq/qis/qubit_qis.h

```diff
@@ -11,7 +11,6 @@
 #include "common/MeasureCounts.h"
 #include "cudaq/qis/modifiers.h"
 #include "cudaq/qis/qarray.h"
-#include "cudaq/qis/qreg.h"
```

Comment:
I know this is already merged, but ...
While this particular change eliminates the warnings in our build, it will also break any user code that is using `qreg` (as evidenced by #1039), unless they specifically add this `#include` into their source code. So, as written, I think this is a breaking change. We may need to add additional documentation in the release notes that specifies that additional `#include`s are needed if the user wants to use `qreg` with our next release.

Reply:
> I know this is already merged, but ... While this particular change eliminates the warnings in our build, it will also break any user code that is using `qreg` (as evidenced by #1039), unless they specifically add this `#include` into their source code. So, as written, I think this is a breaking change. We may need to add additional documentation in the release notes that specifies that additional `#include`s are needed if the user wants to use `qreg` with our next release. Another option would be to add this line back in before our 0.6 release.

#1042 contains a possible alternative to try to keep the best of both worlds. Let me know what you think.